docs/source/usersguide/input.rst

```diff
@@ -292,8 +292,8 @@ OpenMC can use it for on-the-fly Doppler-broadening of resolved resonance range
 cross sections. If this element is absent from the settings.xml file, the
 :envvar:`OPENMC_MULTIPOLE_LIBRARY` environment variable will be used.
 
-  .. note:: The :ref:`temperature_method` must also be set to "multipole" for
-            windowed multipole functionality.
+  .. note:: The <use_windowed_multipole> element must also be set to "true"
```

Comment:
If you're like me, you might have run into trouble with the `JAVA_HOME` environment variable being set to an old version.